boss-ws/src/main/java/org/genomebridge/boss/http/objectstore/S3ObjectStore.java

```diff
@@ -18,16 +18,10 @@
     public S3ObjectStore( ObjectStoreConfiguration config ) {
         bucket = config.bucket;
 
-        if ( config.username != null && config.password != null ) {
-            AWSCredentials creds = new BasicAWSCredentials(config.username, config.password);
-            client = new AmazonS3Client(creds);
-        } else {
-            client = new AmazonS3Client();
-        }
-        if ( config.endpoint != null ) {
-            client.setEndpoint(config.endpoint);
-        }
-        if ( config.pathStyleAccess != null && config.pathStyleAccess ) {
+        AWSCredentials creds = new BasicAWSCredentials(config.username, config.password);
```

Comment:
We now have @NotNull marks on all ObjectStoreConfiguration properties, so the null testing is superfluous.  This old code (no username or password) was to support using an Amazon S3 bucket from "secret" credentials in a local .aws file.  Now that we have a mock object store people don't need any of this extra stuff to run unit tests, and that's better because we don't have to explain anything about how to set up to run unit tests -- they just run.